slmschimp.py

```diff
@@ -812,10 +812,14 @@ def send_to_topic(raw_message):
 
     @staticmethod
     def send_private_msg(subject, message):
+        """
```

Comment:
Code comments are for the living :)
I like to leave comments, especially on any code that doesn't make sense at first blush or might contain a surprise, or a "footgun". Once this PR has merged, we won't care that the old property is no longer supported I think.

Reply:
Sure, will take it out - just wanted to be extra_redundant. :-)